src/NewHoursReport.java

```diff
@@ -1,131 +1,487 @@
+	 		createJFrame(topPane, bottomPane, 300,150);
+	 		frame.pack();
+	 		
+	 	return frame;
+	 		
+	 	}
+	 	
+	 	public static JButton createButton(int width, int height, String name) {
+	 		JButton button = new JButton();
+	 		button.setSize(width, height);
+	 		button.setText(name);
+	 		return button;
+	 	}
+	 	
+	 	public static JFrame createJFrame(JPanel topPane, JPanel bottomPane, 
+	 			int width, int height) {
+	 		JFrame frame = new JFrame();
+	 		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+	 		frame.add(topPane, BorderLayout.PAGE_START);
+	 		frame.add(bottomPane, BorderLayout.PAGE_END);
+	 		frame.setSize(width, height);
+	 		frame.setVisible(true);
+	 		return frame;
+	 	}
+	 	
+	 	public static JFrame changeDatePane() {
+	 		JFrame dateFrame = new JFrame("Change Dates");
+	 		dateFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+	 		
+	 		
+	 		nextListenerClass listener3 = new nextListenerClass();
+	 		backListenerClass listener4 = new backListenerClass();
+	 		//Buttons
+	 		JButton next = createButton(40,40,"Next");
+	 		JButton back = createButton(40,40,"Back");
+	 		next.addActionListener(listener3);
+	 		back.addActionListener(listener4);
+	 		
+	 		//Create Labels
+	 		JLabel start = new JLabel("Start Date");
+	 		JLabel end = new JLabel("End Date");
+	 		
+	 		//TextFields
+	 		 startDate = new JTextField("mm/dd/yyyy", 10);
+	 		 endDate = new JTextField("mm/dd/yyyy", 10);
+	 		
+	 		//Create Panes and Layouts
+	 		JPanel leftPane = new JPanel();
+	 		leftPane.setLayout(new BoxLayout(leftPane, BoxLayout.Y_AXIS));
+	 		JPanel rightPane = new JPanel();
+	 		JPanel bottomPane = new JPanel();
+	 		bottomPane.add(back);
+	 		bottomPane.add(Box.createHorizontalGlue());
+	 		bottomPane.add(next);
+	 		
+	 		//add components to pane
+	 		rightPane.setLayout(new BoxLayout(rightPane, BoxLayout.Y_AXIS));
+	 		leftPane.add(start);
+	 		leftPane.add(Box.createVerticalGlue());
+	 		leftPane.add(startDate);
+	 		
+	 		rightPane.add(end);
+	 		rightPane.add(Box.createVerticalGlue());
+	 		rightPane.add(endDate);
+	 		
+	 		
+	 		//add everything to pane
+	 		dateFrame.add(bottomPane, BorderLayout.PAGE_END);
+	 		dateFrame.add(leftPane, BorderLayout.WEST);
+	 		dateFrame.add(Box.createHorizontalGlue());
+	 		dateFrame.add(rightPane, BorderLayout.EAST);
+	 		dateFrame.pack();
+	 		dateFrame.setVisible(true);
+	 		
+	 		return dateFrame;
+	 	}
+	 	
+	 	public static JFrame employees(ArrayList<Object> names) {
+	 		
+	 		
+	 		JPanel pane = new JPanel();
+	 		pane.setLayout(new BoxLayout(pane,BoxLayout.X_AXIS));
+	 		for (int i = 0; i < names.size();i++) {
+	 			JCheckBox box = new JCheckBox((String)names.get(i));
+	 			pane.add(box);
+	 		}
+	 		
+	 		
+	 		JFrame newFrame = new JFrame("Select Employees");
+	 		newFrame.setDefaultCloseOperation(EXIT_ON_CLOSE);
+	 		newFrame.add(pane);
+	 	
+	 		newFrame.setVisible(true);
+	 		
+	 		return newFrame;
+	 	}
+	 	
+	 public static List<Date> setDate(String start, String end) 
```

Comment:
This is very confusingly spaced. I would suggest having the method definition and throws all on one line. Its also not indented the same as other methods.